modules/core/src/shaderlib/project/project.glsl.js

```diff
@@ -134,6 +134,11 @@ vec4 project_position(vec4 position, vec2 position64xyLow) {
   // METER_OFFSETS or IDENTITY
   // Apply model matrix
   vec4 position_modelspace = project_uModelMatrix * position;
+  if (project_uCoordinateSystem == COORDINATE_SYSTEM_IDENTITY) {
+    position_modelspace.xy -= project_uCoordinateOrigin;
```

Comment:
Maybe add a comment about what these subtractions do, or rather why we do them?

As a complete side note, scanning this, I don't remember what `project_uCoordinateOrigin` is, if it is auto calculated or supplied. 

No need to provide an explanation, I am just pointing out how easy (or not) this code is to read.

Reply:
So this would more correctly be  `position_world.xy -= project_uCoordinateOriginWorld;`?

Reply:
Yes.